x-pack/plugins/canvas/public/components/expression/expression.tsx

```diff
@@ -105,6 +108,23 @@ export const Expression: FC<Props> = ({
     }
   };
 
+  const onEditorDidMount: OnExpressionInputEditorDidMount = (editor) => {
+    /*
+      To enable the CMD+ENTER keybinding, which is running the expression,
+      it is necessary to disable the `-editor.action.insertLineAfter`,
+      which has the same keybinding in the Monaco editor.
+      The only available way is adding the empty dynamic keybinding
+      (by using private monaco API, proposed by the monaco team), which is bubbling the event.
+    */
+    // @ts-expect-error
+    editor?._standaloneKeybindingService.addDynamicKeybinding(
+      '-editor.action.insertLineAfter',
+      // eslint-disable-next-line no-bitwise
+      2048 /* CtrlCmd */ | 3 /* Enter */,
```

Comment:
Is there a reason you aren't using `monaco.KeyMod.CtrlCmd | monaco.KeyCode.Enter` here? Is it to avoid importing monaco in Canvas as well as in Presentation_util?

Reply:
@ThomThomson, thanks for the suggestion, there is no reason for that. We are already importing `Monaco` in some places of Canvas. 

---

Yes, not so long time ago I introduced the PR, which is adding some keybindings for `Monaco`: https://github.com/elastic/kibana/pull/132914. But missed this regression.

So, to remove the regression it is necessary to provide these changes.